executor/write_test.go

```diff
@@ -2040,119 +2040,91 @@ func (s *testSuite4) TestLoadDataOverflowBigintUnsigned(c *C) {
 
 func (s *testSuite4) TestBatchInsertDelete(c *C) {
 	originLimit := atomic.LoadUint64(&kv.TxnEntryCountLimit)
-	defer func() {
-		atomic.StoreUint64(&kv.TxnEntryCountLimit, originLimit)
-	}()
 	// Set the limitation to a small value, make it easier to reach the limitation.
-	atomic.StoreUint64(&kv.TxnEntryCountLimit, 100)
+	var limit4Test uint64 = 2000
+	atomic.StoreUint64(&kv.TxnEntryCountLimit, limit4Test)
```

Comment:
Would this cause unexpected results for other tests running in parallel?

Reply:
There may still be an impact on other tests, but
1. this value will be set back soon now
1. the value itself has been changed to 2000, so it is not easy to reach its limit in other cases.

Reply:
Or we can put this test case into executor/seqtest to run it sequentially?